### PR TITLE
Add `ampersand` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Alts:
Artboard 1@2x

Untitled

<br/>
<br/>

Artboard 1@2x

<br/>

Untitled

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
The large one should match the existing text icons in height, and I'd prefer the bottom to be bulkier than the top.

We could also explore [different ampersand shapes](https://ku-viscom.com/TypeSystems/TypeSystems_TheAmpersand.html#:~:text=There%20are%20two%20basic%20forms,addition%20of%20curves%20and%20flourishes.), maybe some that are closer in shape to `et`, although I'm not sure any of those will actually work out.

**Comment by danielbayley** [comment]
> The large one should match the existing text icons in height

Yeah, good call…

> and I'd prefer the bottom to be bulkier than the top.

@karsa-mistmere It was, but now made it more so…

Also, this is the optimises `path`:
~~~ svg
<path d="m19 13-6.2 5.8a4 4 0 0 1-5.7-5.7l4.6-4.9a2.42 2.42 0 1 0-3.5 0L19 19" />
~~~
But again, it moves everything slightly off grid…

**Comment by karsa-mistmere** [comment]
I'm not really a fan of the shape you went with, it's a bit too sterile for my taste, I'd much rather have something a bit more fluid and playful, more akin to these:

ampersand ampersands

**Comment by danielbayley** [comment]
> more fluid and playful

Ok, see latest `push`, based on these…

Artboard 1@2x

<br/>

Untitled

File: icons/ampersand.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M17.5 12c0 4.4-3.6 8-8 8C7 20 5 18 5 15.5c0-6 8-4 8-8.5 0-1.7-1.3-3-3-3S7 5.3 7 7c0 3 2.5 8.5 12 13" />
+  <path d="M16 12h3" />
```
Comment (karsa-mistmere) [inline icons/ampersand.svg:13]:
```suggestion
  <path d="M17.5 12c0 4.4-3.6 8-8 8A4.5 4.5 0 0 1 5 15.5c0-6 8-4 8-8.5a3 3 0 1 0-6 0c0 3 2.5 8.5 12 13" />
  <path d="M16 12h3" />
```

**Review by karsa-mistmere — APPROVED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**